templates/navbar-user.html

```diff
@@ -0,0 +1,25 @@
+<!-- navbar-user.html -->
+{{ define "navbar-user" }}
+<div class="navbar-reg">
+    <div class="dropdown">
+        <div class="title-plus-arrow">
+            <a class="n-right" href="#">{{ .Extra.UserName }}</a>
```

Comment:
{{ .Extra.UserName }} shows nothing
![empty string](https://github.com/FirstDayAtWork/spfy-app-pages/assets/100528867/66f05f00-27cd-4f67-bbe8-3733a769e7ff)

Reply:
before and after login

Reply:
Because I don't send it. Not yet implemented.

Reply:
Implemented, take a look and close this comment if it works for you too!

Reply:
Everything is fine! Closed